AsyncDisplayKit/ASCellNode.h

```diff
@@ -62,6 +90,17 @@
 - (void)touchesEnded:(NSSet *)touches withEvent:(UIEvent *)event ASDISPLAYNODE_REQUIRES_SUPER;
 - (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event ASDISPLAYNODE_REQUIRES_SUPER;
 
+/**
+ * Marks the node as needing layout. Convenience for use whether the view / layer is loaded or not. Safe to call from a background thread.
```

Comment:
It's only safe to call from the background if the node isn't loaded, right?  We should probably just remove that.